### ISSUE TITLE
LOCALIZATION REQUEST:  Min Dong Chinese (Eastern Min, Foochowese, 閩東語, 福州話) language

### ISSUE THREAD

**Issue by DingSiuyo** [common-voice/common-voice#3787]
# Welcome to the Common Voice Community !

> Common Voice aims to make speech technology accessible to everyone by building an open sourced dataset of labelled voice data that is representative of languages, variants and accents spoken across the world. This template helps us to know how your language could participate in the Common Voice Project. There are three sections of this form, once you have filled out a section please click the checkbox. If you have any issues please contact <commonvoice@mozilla.org>.

## Pontoon Set-up

> To start a language on Common Voice volunteers localise our platform [via Pontoon](https://pontoon.mozilla.org/projects/common-voice/) and create sentence corpus’ of cc0 text. </p>

**Language name**
Min Dong Chinese (per ISO 639-3)
(a.k.a. Eastern Min, Foochownese, 福州话 per Wikipedia)

**Language code**
cdo

**Language size**
10 million (according to Wikipedia, but we thlink the actual number could be way lower)

**Plural forms**

0 rocks 无石头 (no rocks)
1 rock 蜀粒石头
2 rocks 两粒石头
3 rocks 三粒石头
4 rocks 四粒石头
5 rocks 五粒石头
10 rocks 十粒石头
20 rocks 廿粒石头
100 rocks 蜀百粒石头
1000 rocks 蜀千粒石头
I see 0 rocks on the ground 我无看见地兜𠲥有石头
I see 1 rock on the ground 我看见地兜𠲥有蜀粒石头
I see 10 rocks on the ground 我看见地兜𠲥有十粒石头
I see rocks on the ground 我看见地兜𠲥有石头
-->

**Pontoon manager**
https://pontoon.mozilla.org/contributors/ztl8702/

**Language Script**
In terms of "Scripts" as defined by [CLDR](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/languages_and_scripts.html): 
Simplified, Tranditional, Latn

Explantions:
- Simplified, Tranditional: Chinese characters (Hanzi； 汉字) is the most widely used writing system for Min Dong Chinese, similar to the case of Cantonese. The Simplified vs Tranditional distinction is not so clear for Min Dong Chinese, as there is no widespread standarization effort.
- Latin: The following Latin-based scripts are used by Min Dong Chinese
  - [YngPing (榕拼)](https://go.ydict.net/yngping-spec) - A modern romanization scheme for Min Dong Chinese.
  - [Foochow Romanized](https://en.wikipedia.org/wiki/Foochow_Romanized) - A historical writing system invented by Western missionaries to Min Dong region in 1800s, but is now rarely known by its speakers. cdo.wikipedia.org has about 50% of pages written in Foochow Romanized. 

## Sentence Collection Requirements

> On the Common Voice Platform contributors on the platform read out public domain sentences generated through sentence collection. Sentence collection is a crucial part in launching languages on Common Voice. To support the equitable participation of languages of Common Voice we have introduced [three new sentence collection requirements bands](https://discourse.mozilla.org/t/share-your-views-nuancing-sentence-collection-requirements-new-sentence-collection-bands/93134).

**Sentence Requirement Band**

- [X] Band A

- [ ] Band B

- [ ] Band C

## Creating Community 

1. [optional to share] Why do you want to take part in Common Voice ?
  - We would like to learn how to build speech tech (ASR, TTS) for our language, and curate the necessary dataset for training such tools. Mozilla Common Voice is a vibrant community with infrastructure that we can build on top of, and examples we can learn from.
2. [optional to share] Would you like to have a follow up conversation regarding community building ?
  - Yes.

**Comment by ztl8702** [comment]
@Heyhillary Friendly ping. Can we proceed with this request please?

**Comment by ftyers** [comment]
Hi @ztl8702 thanks for pinging. Has a decision been made with regard to orthography / script ?

**Comment by laubonghaudoi** [comment]
@ftyers I talked to @ztl8702 previously and I think the decision was to use Chinese characters as the script.

**Comment by ztl8702** [comment]
Correct. It seems to me that only Chinese-character based script (although unstandardized) is practical.  You will run into ambiguity and Min Dong speakers will disagree over the pronunciation of characters. They might be influenced by Mandarin pronunciation to varying degrees. But I would say most native speakers can get 80% "correct" without training.

Latin-based script is unusable without dedicated prior training, which may be feasible in proprietary projects with hired speakers but not with a public-participation scheme like Common Voice.

**Comment by ftyers** [comment]
Dear @ztl8702, I have set up [Common Voice for Eastern Min](https://pontoon.mozilla.org/cdo/common-voice/) on Pontoon, our localisation platform. Please feel free to go and add some translation suggestions and I will set you up as a translator. If you have any questions, feel free to contact us on [Matrix](https://app.element.io/#/room/#common-voice:mozilla.org).

**Comment by laubonghaudoi** [comment]
That's amazing, thank you so much @ftyers ! @ztl8702 next step for us will be start translating the website into written Foochowese.

I have another question for @ftyers . How to apply to become a translator admin , i.e. getting the power of approving / rejecting a translation?

**Comment by ftyers** [comment]
This is now online!  Congratulations everyone!

**Comment by laubonghaudoi** [comment]
Huge thanks for making Eastern Min on Common Voice! Now that https://commonvoice.mozilla.org/cdo is out, I have some questions:
1. Why is the website translated into Mandarin instead of Eastern Min?
2. Why are all sentences in Mandarin, not Eastern Min sentences at all?
3. Is there an owner for this CDO? There are currently 2 widely used orthographies for CDO, Latin and Chinese characters. Which orthography is picked for this locale? What was the decision process?

**Comment by jessicarose** [comment]
Thanks so much for getting in touch, these are great questions. Eastern Min was added through an accelerated process to facilitate in-field data collection work using the Spontaneous Speech beta. Part of this work involved using the existing Mandarin localization for the website and imported sentences, with plans to appropriately localize the platform and build out a text corpus with appropriate orthographies available after field collection work was completed.
These choices were made with the support of a regional contractor working to support in-field data collection and related platform work.

**Comment by laubonghaudoi** [comment]
Hi @jessicarose , thanks for the info! May I know who this "regional contractor" is? Because CDO was originally requested by @DingSiuyo and @ztl8702 , the team of www.ydict.net/ which should be the owner of this language split. However, the site right now is a copy of zh-tw, which I don't think is the original intention of @ztl8702  and @DingSiuyo . I suspect that this "regional contractor" is not from their team and this entire thing is a miscommunication. It seems to me like there are different groups of people trying to push the same thing towards different directions, without knowing each others' existence.

**Comment by jessicarose** [comment]
Thanks so much for flagging this, I'll try and reach out to the original localization requestors and pick this up with them, thank you!

**Comment by laubonghaudoi** [comment]
Hi @jessicarose , the contacts are only on WeChat and they don't have emails. Do you have a WeChat account?

**Comment by jessicarose** [comment]
We’re happy to remove the placeholder localisation altogether, as it was only meant to be a fallback in case a new platform we had in beta-testing didn’t function as expected during a planned event. The event is now over, and we’re hoping to release that small Spontaneous Malaysian Fuzhou dataset in 2025!

We’re also happy to try and connect the various different Fuzhou communities and co-ordinators via email if everybody would like to chat - WeChat is challenging as we don’t have it and neither do all the participants. In the meantime, we’ll remove this and await preferred CDO sentences and strings from @laubonghaudoi and other community members working on standardising CDO orthography. Thank you again for everyone involved for your interest in the language and for getting it up and running on Common Voice!

**Comment by laubonghaudoi** [comment]
I see. Can you elaborate on this "Spontaneous Malaysian Fuzhou" a bit more? I am surprised that someone is going to make a Malaysian Fuzhou because the team of @ztl8702 was gonna only focus on Fuzhou in mainland China, and AFAIK there doesn't exist a Malaysian Fuzhou dialect. Are you mistaking Malaysian Hokkien as Fuzhou? Or is this Malaysian Fuzhou dialect requested from some other groups?

If we are to remove the placeholder localization, then there will be a translation manager who approves all those translation strings. May I know who this manager is? We will need to discuss with them about the translation strategy for this website to better collaborate. Thanks!

**Comment by ftyers** [comment]
I'm closing this one as there is a separate conversation ongoing.
